hikari/api/entity_factory.py

```diff
@@ -143,6 +143,21 @@ def deserialize_own_guild(self, payload: data_binding.JSONObject) -> application
             The deserialized "own guild" object.
         """
 
+    @abc.abstractmethod
+    def deserialize_invite_application(self, payload: data_binding.JSONObject) -> application_models.InviteApplication:
```

Comment:
Don't think we need a dedicated method for deserializing this since it's only ever found on invite objects